cfme/common/__init__.py

```diff
@@ -702,9 +702,17 @@ class CustomButtonEventsView(View):
 
     @property
     def is_displayed(self):
+        obj = self.context["object"]
+
+        if isinstance(obj.parent,
+                      self.browser.appliance.collections.object_store_containers.__class__):
```

Comment:
@digitronik is there a shorter way to get this `Class`? A shorter call would improve readability.

Reply:
@john-dupuy there isn't really, because we're in a View class method, and need to get access to the appliance.

That being said, @digitronik its not really clear why this check is being made, what the purpose is here. Needs some in-line comments please.

Reply:
@john-dupuy: I agreed with you. I had two option that import class from storage model and second one collection. but now days, I love to use collection possibilities. I added comment.

@mshriver let me know if you found that i should go with first option. I will add change. :+1:

Reply:
@digitronik thanks for adding comments here - This seems an odd lookup to me, that could easily break when the parent collection of an ObjectStoreContainer is not an ObjectStoreContainerCollection.

This could quickly grow to having to check multiple collection types, parent types, etc if the Entity doesn't have the 'name' attribute/property.

I think its reasonable to state in the docblock for `CustomButtonEventsMixin` that a `name` attribute/property is required to use the navigation/view provided by the mixin.  Classes inheriting it can then define that - in this case, `ObjectStoreContainer.name` could be a property that returns the key attribute.

Reply:
Done... added property.